### PR TITLE
Fixes an Icon issue with the walls on research and jobdef/bloodpacks

### PR DESCRIPTION
**Pull request by Flaksim**
- Added blood pack crates to medical bays.
- fixed an issue with wall icons on research, Lower level is now reinforced walls.
- Fixes Depot Guard job.

### CODE REVIEW

File: maps/Asteroid Listening Post/spawns_jobs.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@ GLOBAL_LIST_EMPTY(listen_staff_spawns)
 	hierarchy_type = /decl/hierarchy/outfit/job
 
 
-/datum/job/IGUARD
+/datum/job/LISTENG
```
Comment (DoctorOctopusTheOctopi) [inline maps/Asteroid Listening Post/spawns_jobs.dm:36]: Did you mean to replace this?

Comment (Flaksim) [reply maps/Asteroid Listening Post/spawns_jobs.dm:36]: I did, it was a duplicate definition, IGUARD was intended for depot guard, not listening post technician.

Comment (DoctorOctopusTheOctopi) [reply maps/Asteroid Listening Post/spawns_jobs.dm:36]: Berry good.

Comment (X0-11) [reply maps/Asteroid Listening Post/spawns_jobs.dm:36]: shoddy copy paste work you have there :-1:

**Review by DoctorOctopusTheOctopi — APPROVED** [review]
Small changes nothing gamebreaking, looks good. 👍 🐙

**Pull request merged**